includes/class-sensei-settings.php

```diff
@@ -39,6 +39,8 @@ public function __construct() {
 		$this->register_hook_listener();
 		$this->get_settings();
 
+		// Log when settings are updated by the user.
+		add_action( 'update_option_sensei-settings', [ $this, 'log_settings_update' ], 10, 2 );
```

Comment:
I understand why there are mixed underscores and dashes in the hook, but I think it will be easier for developers to use these filters if we stick to using just underscores.

Reply:
So just to clarify, is your suggestion to make sure that our option names have underscores instead of slashes from now on?

We could also change the existing option names with slashes if desired. We should do that in a new PR.

Reply:
Oops. Ignore this please. I read this as a `do_action` and thought it was a new action being added by us. I guess I should stay out of the code for reals!

Reply:
Ah that makes sense. No worries! Thanks for mentioning, and clarifying! 🙂